docker-compose.yml

```diff
@@ -28,7 +30,7 @@ services:
             CHAIN_ID: '${CHAIN_ID}'
             POSTGRES_URI: 'postgres://api:api@postgres/api'
     mesh:
-        image: 0xorg/mesh:10.2.0
+        image: overmorrow/mesh:v4rc4
```

Comment:
Can we update the tag if there is a "official" image tag now?

Reply:
Yep, we will publish a proper version after we are done testing on staging 👍